frontend/javascripts/admin/voxelytics/task_list_view.tsx

```diff
@@ -421,6 +425,26 @@ export default function TaskListView({
     }
   }
 
+  async function deleteWorkflowReport() {
+    await modal.confirm({
+      title: "Delete Workflow Report",
+      content:
+        "Are you sure you want to delete this workflow report?\nNote that if the workflow is still running, this may cause it to fail.",
```

Comment:
```suggestion
        "Are you sure you want to delete this workflow report? This can not be undone.\nNote that if the workflow is still running, this may cause it to fail.",
```